### PR TITLE
Review "Backport changes" page

### PR DESCRIPTION
**Pull request by jdbaldry**
Closes https://github.com/grafana/writers-toolkit/issues/495.

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/review/backport-changes/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+weight: 500
+---
+
+# Backport changes
+
+Projects with versioned documentation typically maintain each version with the code in multiple long-lived branches.
+The `main` branch has the most recent code and documentation.
+
+Project releases typically use long-lived branches that include the major and minor versions of the release.
+For example in the `grafana/grafana` repository, the `v9.0.x` long-lived release branch contains code and documentation for all patched versions of the major version `9` and minor version `0` of Grafana.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:22]:
```suggestion
For example, in the `grafana/grafana` repository, the `v9.0.x` long-lived release branch contains code and documentation for all patched versions of the major version `9` and minor version `0` of Grafana.
```

File: docs/sources/review/backport-changes/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Backporting takes a change from the `main` branch, and ports it back to another long-lived release branch.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:24]:
```suggestion
Backporting takes a change from the `main` branch and ports it back to another long-lived release branch.
```

File: docs/sources/review/backport-changes/index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Every documentation pull request to a project with versioned documentation requires a decision about whether or not to backport it.
+
+You should backport to all supported versions of the project affected by the pull request changes.
+
+## Backport a change
+
+To backport a change, add the appropriate `backport <BRANCH>` label.
+You can backport to more than one branch by using multiple labels.
+
+After the pull request is merged, Grot, the Grafana bot, creates a follow-up pull request for each of the `backport <BRANCH>` labels.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:35]:
```suggestion
After you merge the pull request, Grot, the Grafana bot, creates a follow-up pull request for each of the `backport <BRANCH>` labels.
```

File: docs/sources/review/backport-changes/index.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+If Grot is unable to automatically backport the changes, it comments on the first pull request with instructions about how to backport the change manually.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:36]:
```suggestion
If Grot can't automatically backport the changes, it comments on the first pull request with instructions about how to backport the change manually.
```

File: docs/sources/review/backport-changes/index.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+In repositories such as `grafana/grafana`, engineers sometimes create a branch for a release well before the release has shipped.
+If you intend to publish content against an imminent release, check for the existence of a backport label for the upcoming version before merging the pull request.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:39]:
```suggestion
If you intend to publish content against an imminent release, check for a backport label for the upcoming version before merging the pull request.
```

File: docs/sources/review/backport-changes/index.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  D3 -- No -->  D4
+  D4 -- Yes --> T4
+  D4 -- No -->  T3
+</div>
+
+<!-- vale Grafana.Timeless = YES -->
+
+## Backporting tutorials
+
+Depending on the location from which the website mounts tutorials content, you might need to backport changes to them.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:78]:
```suggestion
Depending on the location from which the website mounts tutorial content, you might need to backport changes to them.
```

File: docs/sources/review/backport-changes/index.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+For `grafana/grafana`, the website mounts tutorials content from the `next` version of documentation, so you don't need to backport changes.
```
Comment (clayton-cornell) [inline docs/sources/review/backport-changes/index.md:80]:
```suggestion
For `grafana/grafana`, the website mounts tutorial content from the `next` version of the documentation, so you don't need to backport changes.
```

**Review by clayton-cornell — COMMENTED** [review]
Some minor tweaks

**Comment by jdbaldry** [comment]
Thanks for the suggestions, they look good to me!

**Review by clayton-cornell — APPROVED** [review]
LGTM

**Pull request merged**